play/materialize-cloud-dbt/models/example/market_orders.sql

```diff
@@ -0,0 +1,32 @@
+-- Copyright Materialize, Inc. and contributors. All rights reserved.
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License in the LICENSE file at the
+-- root of this repository, or online at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+{% set source_name %}
+    {{ mz_generate_name('market_orders_raw') }}
+{% endset %}
+
+{% set stmt %}
+    CREATE MATERIALIZED SOURCE {{ source_name }} FROM PUBNUB
+    SUBSCRIBE KEY 'sub-c-4377ab04-f100-11e3-bffd-02ee2ddab7fe'
+    CHANNEL 'pubnub-market-orders';
+{% endset %}
+
+{{ mz_drop_source(source_name, if_exists=True, cascade=True) }}
+{{ mz_add_source(stmt) }}
```

Comment:
I think this should be `mz_create_source`?

Reply:
Thank you!